carmajava/cooperativemerge/src/main/java/gov/dot/fhwa/saxton/carma/plugins/cooperativemerge/StandbyState.java

```diff
@@ -48,7 +48,14 @@
   protected final static int LOOP_SLEEP_TIME = 1000; // ms
 
   protected final static double CM_PER_M = 100.0;
-  
+
+  protected final static double MIN_RADIUS = 100.0; //meters
+  protected final static double MAX_RADIUS = 10000.0; //meters
+  protected final static double MIN_MERGE_DIST_FROM_METER = 0.0; //meters
```

Comment:
You probably want the MIN_MERGE_DIST_FROM_METER to go to -10 to allow for some buffer room around the meter point as the units are in downtrack distance.

Reply:
done